### ISSUE TITLE
Chapter outline font isn't legible on Story Maps

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-web-client#2455]
## Description
The text color for the chapter outline is too low in contrast to meet accessibility guidelines or be legible. The current color is #1D438A. It should be changed to #358EAE.

![image](https://github.com/user-attachments/assets/5ed11db5-ece0-460a-b955-ba5e3a4ab97d)

**Comment by paulschreiber** [comment]
I could not find `#1D438A ` anywhere. I changed `#1671bc` to ` #358eae`.
